jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -429,16 +439,30 @@ def _image_viewer_update(self, viewer, x, y):
 
         if (-0.5 < x < image.shape[ix_shape] - 0.5 and -0.5 < y < image.shape[iy_shape] - 0.5
                 and hasattr(active_layer, 'attribute')):
+
             attribute = active_layer.attribute
+
             if isinstance(viewer, (ImvizImageView, MosvizImageView, MosvizProfile2DView)):
                 value = image.get_data(attribute)[int(round(y)), int(round(x))]
+                if associated_dq_layer is not None:
+                    dq_attribute = associated_dq_layer.state.attribute
+                    dq_data = associated_dq_layer.layer.get_data(dq_attribute)
+                    dq_value = dq_data[int(round(y)), int(round(x))]
                 unit = image.get_component(attribute).units
             elif isinstance(viewer, CubevizImageView):
                 arr = image.get_component(attribute).data
                 unit = image.get_component(attribute).units
                 value = self._get_cube_value(image, arr, x, y, viewer)
             self.row1b_title = 'Value'
-            self.row1b_text = f'{value:+10.5e} {unit}'
+
+            if associated_dq_layer is not None:
+                if np.isnan(dq_value):
+                    dq_text = ''
+                else:
+                    dq_text = f' (DQ: {int(dq_value):d})'
+            else:
+                dq_text = ''
+            self.row1b_text = f'{value:+10.5e} {unit}{dq_text}'
```

Comment:
This is the simplest and fastest possible DQ injection into coords-info that I could do. We could also decompose the bit flag, but that is (1) hard to fit in the horizontal space, and (2) a tiny bit more expensive.